### PR TITLE
allow latest pkg:pub_semver

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
Feel free to squash/merge and delete the branch if you're happy

**Review by natebosch — COMMENTED** [review]
Need to change the version in `pubsepc.yaml`. Add a `-dev` suffix unless you plan on publishing immediately.

File: pubspec.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ dependencies:
   checked_yaml: ^1.0.0
   # Verified that no new features since 1.0.0 are used - be careful!
   json_annotation: '>=1.0.0 <5.0.0'
-  pub_semver: ^1.3.2
+  pub_semver: ^2.0.0
```
Comment (natebosch) [inline pubspec.yaml:15]: How about a range for now?

```suggestion
  pub_semver: ">=1.3.0 <3.0.0"
```

If you feel we should be using this as a new lower bound, please change the commit message from "allow" to "require"

Comment (kevmoo) [reply pubspec.yaml]: yup. oops. I plan to publish ASAP

**Comment by kevmoo** [comment]
> Need to change the version in `pubsepc.yaml`. Add a `-dev` suffix unless you plan on publishing immediately.

I plan to publish ASAP

**Review by natebosch — APPROVED** [review]

**Pull request merged**